templates/geth/schema.capnp

```diff
@@ -0,0 +1,13 @@
+@0xb96c7457b3a2bc76;
+
+struct Schema {
+    network @0: Text="testnet"; # ethereum network preference, 'testnet', 'rinkeby', ..'
+    lightserv @1: UInt32=90;     # ethereum lightserv
+    verbosity @2: UInt32=4;     # ethereum logging
+    v5disc @3: Text; # ethereum v5disc packets
+    syncmode @4: Text="full"; # ethereum network syncmode, 'full', 'fast', ..
+    nat @5: Text="none"; # ethereum natmode, 'none', ..
+    datadir @6: Text="/mnt/data" Text; # ethereum datadir
```

Comment:
This does not look right, duplicate Text?

Reply:
fixed in https://github.com/threefoldfoundation/tfchain-templates/pull/3/commits/fea503883994a25d6d26e5cc84cebdfa19afa514